src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
+import BooksList from './Components/BooksList';
+import Navbar from './Components/Navigation';
+import Books from './Components/BooksState';
+
+function App() {
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <Navbar />
+
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/author" element={<BooksList />} />
+        </Routes>
```

Comment:
- [ ] Kindly make sure to create a `<Categories>` component so that the second route redirects to that component instead of `<BooksList />` component.